.github/ISSUE_TEMPLATE.md

```diff
@@ -1,26 +1,8 @@
 <!--
 
 	Note that we use GitHub issues for bugs and (uncontroversial) feature
-	requests (see below for details).
-
-	Please do *NOT* ask usage questions in GitHub issues.  Usage questions make
-	most sense on the users mailing list, where more people are available to
-	potentially respond to your question, and the whole community can benefit
-	from the answers provided (perhaps your question has already been answered,
-	search the archive to find out):
-	https://groups.google.com/forum/#!forum/prometheus-users
-
-	While a GitHub issue is fine to track progress on an uncontroversial
-	feature request, many feature requests touch the best practices and
-	concepts of Prometheus as a whole and need to be discussed with the wider
-	developer community first. This is in particular true for a request to
-	reconsider a prior rejection of a feature request or any of the declared
-	non-goals (see README.md). Those overarching discussions happen on the
-	developer mailing list (GitHub issues, in particular closed ones, are not
-	tracked by the wider developer community and thus inadequate):
-	https://groups.google.com/forum/#!forum/prometheus-developers
-	
-	You can find more information at: https://prometheus.io/community/
+	requests. For questions and discussions, please use GitHub discussions or
+	any of the other community channels: https://prometheus.io/community/
```

Comment:
Makes it sound like it is fine to open GitHub discussions for questions. I guess that is not what we want?

Reply:
yes

Reply:
Not sure what is that `yes` for since you also approved the PR.

Reply:
Oh I see Q&A in Discussions now :) I get your "yes" now